widget/iframe/src/styles.ts

```diff
@@ -1,7 +1,19 @@
-export const commonStyles = {
-  width: '100vw',
-  maxWidth: '390px',
-  height: '100vh',
-  maxHeight: '700px',
-  overflow: 'hidden',
-};
+const commonStyles = `
+  width: 100vw;
+  max-width: 390px;
+  height: 100vh;
+  max-height: 700px;
+  overflow: hidden;
+  background-color: transparent;
+  border: none;
+`;
+
+export function applyDefaultStyles(element: HTMLIFrameElement) {
+  element.style.cssText = commonStyles;
+  element.style.height = '100vh';
+  element.style.maxHeight = '700px';
+}
+export function applyDynamicHeightStyles(element: HTMLIFrameElement) {
+  element.style.backgroundColor = 'transparent';
```

Comment:
Transparent background color is already present in common styles; you can exclude it from here.

Reply:
fixed